library/src/scala/quoted/Quotes.scala

```diff
@@ -3588,21 +3589,19 @@ trait Quotes { self: runtime.QuoteUnpickler & runtime.QuoteMatching =>
         def declaredFields: List[Symbol]
 
         /** Get named non-private fields declared or inherited */
-        // TODO: deprecate in 3.1.0 and remove @experimental from fieldMember
-        // @deprecated("Use fieldMember", "3.1")
+        // TODO: deprecate after 3.1.x
```

Comment:
Why can't they be deprecated immediately now that there's a replacement?

Reply:
Just to give a longer migration period. I could deprecate it in `3.1.0` or `3.1.1` if we want to deprecate it quickly.

Reply:
Normally we should deprecate as soon as there's a way to fix the deprecation warning, no need to delay since people who don't want to migrate now can always ignore warnings.

Reply:
Ok. I will deprecate it.